src/components/github-htmlview.component.js

```diff
@@ -65,6 +65,17 @@ const styles = {
   a: linkStyle,
 };
 
+const quotedEmailToggleStyle = {
+  backgroundColor: colors.greyMid,
+  paddingLeft: 4,
+  paddingRight: 4,
```

Comment:
If use `paddingHorizontal` instead `paddingLeft` and `paddingRight`, will it work?

Reply:
yes, thank you for the reminder 👍